src/pages/app-bar/collapsible-app-bar/index.tsx

```diff
@@ -125,7 +132,7 @@ export const CollapsibleAppBar = (): JSX.Element => {
                 </Toolbar>
             </AppBar>
             <div className={clsx(classes.banner, classes.alignTopContent)}>
-                <Toolbar className={classes.bannerMain} data-cy={'banner'}>
+                <Toolbar className={classes.bannerMain} data-cy={'banner'} style={{ paddingLeft: 16 }}>
```

Comment:
classes={{ gutters: classes.toolbarGutters }}

Reply:
Good catch, this is updated now